src/app/views/datasets-new/edit-metadata-cantabular/CantabularMetadataController.jsx

```diff
@@ -127,13 +133,56 @@ export class CantabularMetadataController extends Component {
         };
     }
 
-    UNSAFE_componentWillMount() {
+    async UNSAFE_componentWillMount() {
         const datasetID = this.props.params.datasetID;
         const editionID = this.props.params.editionID;
         const versionID = this.props.params.versionID;
         this.getMetadata(datasetID, editionID, versionID);
+        await this.getTopics();
     }
 
+    getTopics = async () => {
+        try {
+            let rootTopicsArr = await topics.getRootTopics().then(rootTopics => rootTopics.items);
+            let allSubTopics = [];
+            await Promise.all(
+                rootTopicsArr.map(rootTopic =>
+                    topics.getSubTopics(rootTopic.id).then(subTopics => {
+                        allSubTopics = [...allSubTopics, ...subTopics.items];
+                    })
+                )
+            );
+            const rootTopics = {
```

Comment:
Line before this:

const extractTopicOptions = ({ id, next: { title }}) => ({ value: id, label: title })